nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/scheduler/Schedule.java

```diff
@@ -80,4 +80,13 @@
      */
     boolean isEnabled();
 
+    /**
+     * Return the timezone to be used for the Cron Expression.
+     *
+     * @since 10.2
+     * @return
```

Comment:
`Return` -> `Returns`
`@return` -> `@return the timezone, or {@code null} if not specified`